Drivers/NER/include/pi4ioe.h

```diff
@@ -0,0 +1,73 @@
+/*
+	PI4IOE5V9535ZDEX GPIO Expander Header File
+	Link to part Datasheet for reference:
+	https://www.diodes.com/assets/Datasheets/PI4IOE5V9535.pdf
+
+	Author: Dylan Donahue
+*/
+
+#ifndef PI4IOE_H
+#define PI4IOE_H
+
+#include <stdint.h>
+#include "stm32f4xx_hal.h"
+#include "stm32f4xx_hal_i2c.h"
+
+// all of these addresses are complete bullshit and wrong plz fix when know :) and remove this comment
+// the values should be correct though, just the order is wrong
+
+#define PI4IOE_I2C_ADDR 	0x20 //? @hamza does HAL driver handle shifting for read/write bit automatically? if not should be added here
+
+#define IO_CONFIG_REG 		0x06 
+#define IO_CONFIG_BUF 		0x10 /* = 0001 0000, bit 4 = TSMS = input */ 
```

Comment:
Would it be worth it to expand this into a union to explicitly set each bit? If we use this chip anywhere else it might be nice to have it not be hardcoded